drivers/sensor/iis2iclx/iis2iclx.h

```diff
@@ -49,9 +49,7 @@ struct iis2iclx_config {
 	uint8_t odr;
 	uint8_t range;
 #ifdef CONFIG_IIS2ICLX_TRIGGER
-	const char *irq_dev_name;
-	uint8_t irq_pin;
-	uint8_t irq_flags;
+	const struct gpio_dt_spec gpio_drdy;
```

Comment:
You'd get less wasted padding if you moved the `uint8_t` member above the `gpio_dt_spec` member.

Reply:
Are you suggesting to move ```uint8_t int_pin; ```  before the gpio_dt_spec structure?

Reply:
Yes.

Reply:
Yes, sure. Keeping the uint8_t all together should result in a better structure packing. THanks.